### PR TITLE
chore: remove tqdm

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

closes FL-1669

### CODE REVIEW

File: backend/image_transfer/encoder.py [hunk 1]
```diff
@@ -61,12 +60,10 @@ def download_blob_to_zip(dxf_base: DXFBase, blob: Blob, zip_file: safezip.ZipFil
...
-    with tqdm(total=total_size, unit="B", unit_scale=True) as pbar:
-        blob_path_in_zip = f"blobs/{blob.digest}"
-        with zip_file.open(blob_path_in_zip, "w", force_zip64=True) as blob_in_zip:
-            for chunk in bytes_iterator:
-                blob_in_zip.write(chunk)
-                pbar.update(len(chunk))
+    blob_path_in_zip = f"blobs/{blob.digest}"
+    with zip_file.open(blob_path_in_zip, "w", force_zip64=True) as blob_in_zip:
+        for chunk in bytes_iterator:
+            blob_in_zip.write(chunk)
```
[the remark this answers is not in the document]
Comment (ThibaultFy) [reply backend/image_transfer/encoder.py:66]: ? What do you mean

[the remark this answers is not in the document]
Comment (ThibaultFy) [reply backend/image_transfer/encoder.py:66]: Ah you mean to still have access to some logs ? Sorry, I'll add it

File: backend/image_transfer/encoder.py [hunk 2]
```diff
@@ -61,12 +63,12 @@ def download_blob_to_zip(dxf_base: DXFBase, blob: Blob, zip_file: safezip.ZipFil
     bytes_iterator, total_size = repository_dxf.pull_blob(blob.digest, size=True)
 
     # we write the blob directly to the zip file
-    with tqdm(total=total_size, unit="B", unit_scale=True) as pbar:
-        blob_path_in_zip = f"blobs/{blob.digest}"
-        with zip_file.open(blob_path_in_zip, "w", force_zip64=True) as blob_in_zip:
-            for chunk in bytes_iterator:
-                blob_in_zip.write(chunk)
-                pbar.update(len(chunk))
```
[the remark this answers is not in the document]
Comment (ThibaultFy) [reply backend/image_transfer/encoder.py:69]: Don't know :/ WDYT ?

File: backend/image_transfer/encoder.py [hunk 3]
```diff
@@ -23,6 +23,8 @@
 from substrapp.docker_registry import RegistryPreconditionFailedException
 from substrapp.utils import safezip
 
+logger = structlog.get_logger("events")
```
[the remark this answers is not in the document]
Comment (ThibaultFy) [reply backend/image_transfer/encoder.py:26]: my bad thanks for catching this

File: changes/990.removed [hunk 4]
```diff
@@ -0,0 +1 @@
+`tqdm`dependency, only used in logging
```
Comment (guilhem-barthes) [inline changes/990.removed:1]:
```suggestion
`tqdm` dependency, only used in logging
```

**Review by guilhem-barthes — APPROVED** [review]

**Review by thbcmlowk — APPROVED** [review]
Thank you both, LGTM!

**Pull request merged**